CrashReporter.xcodeproj/project.pbxproj

```diff
@@ -3813,6 +3814,7 @@
 				HEADER_SEARCH_PATHS = "\"$(SRCROOT)/Dependencies/protobuf-c\"";
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				MACOSX_DEPLOYMENT_TARGET = 10.7;
+				OTHER_CFLAGS = "-fembed-bitcode";
```

Comment:
Is there separate setting for it? Let's avoid usage `OTHER_CFLAGS` when possible

Reply:
Changed to BITCODE_GENERATION_MODE.